src/exchangeGroups.js

```diff
@@ -12,15 +12,15 @@ const queueDefaults = {
   noAck: false,
 };
 
-function groupFromInput(param, defaultName) {
+function groupFromInput(param, { name, persistent }) {
   let returnVal = {};
   if (_.isString(param)) {
     returnVal = { name: param };
   } else if (_.isObject(param)) {
     returnVal = param;
   }
-
-  returnVal.name = returnVal.name || defaultName;
+  returnVal.name = returnVal.name || name;
+  returnVal.persistent = returnVal.persistent || persistent;
```

Comment:
it looks like returnVal.persistent is not being assigned so, probably it is not necessary to compare its value..

Reply:
That is to allow if you only want to persist messages in 1 exchange, say only in `rejectedExchange`

Reply:
> That is to allow if you only want to persist messages in 1 exchange, say only in `rejectedExchange`

Yes, my bad